CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Features, Bug Fixes, API Breaking, Deprecated, Infrastructure, Template Updates
 - Add unit tests for all transaction types in runtime executeTx.
 - Add support loadLogic for *.teal program and SCParam.
 - Replace arrow functions with normal functions in all unit test which is recommended by `mocha`
+- Added support for unfunded accounts in `Runtime`.
```

Comment:
Now I think if this should be under Features

Reply:
Yes. It's a feature. I have moved it to feature sections.